com.aobuchow.themes.spectrum.preferences/src/com/aobuchow/themes/spectrum/preferences/PreferencesPage.java

```diff
@@ -109,26 +76,20 @@ private void addHSLCustomization(Group customizeGroup) {
 			colorManager.setBaseColor(newBaseColor.getColor());
 			colorManager.setAccentColor(newAccentColor.getColor());
 			colorManager.saveColors();
+			System.out.println("save hue");
```

Comment:
Remove before merge ;)

Reply:
Ouh nou! 😇 . Yes, those gotto go. Thank you for finding. :)

Reply:
No worries :D

Reply:
Fixed in latest push (amended commit)